api/python/notebooks/experimental/mean_variance.ipynb

```diff
@@ -0,0 +1,726 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "c4b089f9-5304-4479-b739-20792e12595a",
+   "metadata": {},
+   "source": [
+    "# Out-of-core mean and variance calculation\n",
+    "\n",
+    "This tutorial describes use of the cellxgene_census.experimental.pp API for calculating out-of-core mean and variance in the Census. The variance calculation is performed using [Welford's online algorithm](https://en.wikipedia.org/wiki/Algorithms_for_calculating_variance).\n",
+    "\n",
```

Comment:
See [Census API notebook/vignette editorial guidelines](https://github.com/chanzuckerberg/cellxgene-census/blob/main/docs/census_notebook_guidelines.md)

Please add a table of contents:

```
**Contents**

1. The mean and variance API.
2. Example: calculate mean and variance for a slice of Census.
```